contracts/optimistic-ethereum/iOVM/bridge/tokens/iOVM_L1ETHGateway.sol

```diff
@@ -2,48 +2,24 @@
 pragma solidity >0.5.0;
 pragma experimental ABIEncoderV2;
 
+import { iOVM_L1TokenGateway } from "./iOVM_L1TokenGateway.sol";
+
 /**
  * @title iOVM_L1ETHGateway
  */
-interface iOVM_L1ETHGateway {
-
-    /**********
-     * Events *
-     **********/
-
-    event DepositInitiated(
-        address indexed _from,
-        address _to,
-        uint256 _amount
-    );
-
-    event WithdrawalFinalized(
-        address indexed _to,
-        uint256 _amount
-    );
-
+interface iOVM_L1ETHGateway is iOVM_L1TokenGateway {
 
     /********************
      * Public Functions *
      ********************/
 
-    function deposit()
+    function depositETH()
```

Comment:
Using the same function names as `iOVM_L1TokenGateway` would allow us to delete this file...

Reply:
I considered this, but it's a little tricky:
- It would require the ERC20 contract functions be `payable`
- It would require sending `OVM_L1ETHGateway.deposit{value: myValue}(myValue)` where you pass in the value twice.  
For this reason, I went with reverting and having the L1 ETH gateway mostly untouched.  Open to future exploration here; maybe inheriting the events at least?  Overall I feel like inheritance is weird and we shouldn't abuse it, thought.